.github/pull_request_template.md

```diff
@@ -0,0 +1,31 @@
+<!--
+Use this template as a guide. Omit sections that don't apply.
+More details about the template are at https://github.com/edx/open-edx-proposals/pull/180
+-->
+
+## Description
+
+Describe what this pull request changes, and why. Include implications for people using this change.
```

Comment:
If we ask folks to document and duplicate the "why" in the PR description, how do we ensure and clarify that the "why" needs to be documented in docstrings and ADRs?  Instead, should we nudge them to document in ADRs and ask them to link to the "why" in the description?

Reply:
The PR description should have the why of the PR as a whole.  ADRs document design decisions made along the way. Most PRs don't have ADRs.  Docstrings should be focused on the specific function/class/module that they document. I don't think there will be much duplication among these three different places.

Reply:
ADRs document both the decisions and the rationale behind those decisions.

To rephrase my concern: I'd like to remind and encourage folks to adopt ADRs as part of their work. The ADRs provide the historical trace of the changes and their rationale. The adoption of the ADR process is what allows us to not have a central blocking architectural council. Thus, I want to ensure folks do not consider adding docs in PR descriptions as a replacement for ADRs.

Reply:
Do we agree that most PRs will not involve an ADR and that most PRs will want a "why" described?  I wonder if we are starting from different understandings.  I definitely agree that ADR-worthy decisions should be recorded in ADRs, and that there's no need to duplicate that decision process in the PR template.

Reply:
I agree that many PRs do not need an ADR.
I believe the wording below can be adjusted to address the "nudge" that I'm looking for:

`Reminder that design decisions and their rationales are to be documented in the repo (docstring / ADR), per OEP-19, and can be linked here.`